docs/DeveloperGuide.md

```diff
@@ -1027,31 +1028,33 @@ otherwise)
 
 **Actor:** Registered owner
 
-**Preconditions:** Registered owner is logged out.
+**Preconditions:** An owner is registered with HomeBoss.
 
 **Guarantees:**
 
-- Password would be changed.
+- Password will be changed.
 
 **MSS:**
 
 1. Registered owner opens the HomeBoss application.
-2. Registered owner enters the account recovery command without any command flags (i.e., `--answer`).
-3. US displays the forget password question that the user set during account registration.
+2. Registered owner enters the account recovery command without any command flags (i.e., no `--answer`).
+3. US displays the "forget password" secret question that the user set during account registration, together with the
+   command format for account recovery.
```

Comment:
Would this be considered giving too much details about the mechanics of the system, i.e. "--answer"